automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEDataSyncingSpec.scala

```diff
@@ -21,241 +21,244 @@ import scala.concurrent.duration._
-    "User should be able to create files outside of playground and safe mode" in { runtimeFixture =>
-      val fileName = "mockUserFile.ipynb"
+  "Locked by another user and playground mode transition handled correctly" ignore { runtimeFixture =>
+    val sampleNotebook = ResourceFile("bucket-tests/gcsFile4.ipynb")
+    val isEditMode = true
+    val isRStudio = false
 
-      val mockUserFile: File = JupyterServerClient.createFileAtJupyterRoot(runtimeFixture.runtime.googleProject,
-                                                                           runtimeFixture.runtime.clusterName,
-                                                                           fileName
+    withResourceFileInBucket(runtimeFixture.runtime.googleProject, sampleNotebook, "text/plain") { gcsPath =>
+      // we set the lock before the notebook is open to cause a conflict
+      val newMeta = Map("lockExpiresAt" -> Instant.now().plusMillis(20.minutes.toMillis).toEpochMilli.toString,
+                        "lastLockedBy" -> "NotMe"
       )
+      setObjectMetadata(gcsPath.bucketName, GcsBlobName(gcsPath.objectName.value), newMeta).unsafeRunSync
+
+      withWelderInitialized(runtimeFixture.runtime, gcsPath, ".*.ipynb", isEditMode, isRStudio) { localizedFile =>
+        withWebDriver { implicit driver =>
+          withOpenNotebook(runtimeFixture.runtime, localizedFile, 5.minutes) { notebookPage =>
+            eventually(timeout(Span(2, Minutes))) { // wait for checkMeta tick
+              val lockIssueElements =
+                List(notebookPage.lockPlaygroundButton, notebookPage.lockCopyButton, notebookPage.modalId)
 
-      withWebDriver { implicit driver =>
-        withOpenNotebook(runtimeFixture.runtime, mockUserFile, 5.minutes) { notebookPage =>
-          notebookPage.modeExists() shouldBe false
-          notebookPage.areElementsPresent(List(notebookPage.noModeBannerId)) shouldBe true
-          notebookPage.getMode() shouldBe NotebookMode.NoMode
+              notebookPage.areElementsPresent(lockIssueElements) shouldBe true
+
+              notebookPage.executeJavaScript(
+                "window.onbeforeunload = null;"
+              ) // disables pesky chrome modal to confirm navigation. we are not testing chrome's implementation and confirming the modal proves problematic
+              notebookPage.goToPlaygroundModeFromLockIssue
+            }
+
+            eventually(timeout(Span(30, Seconds))) { // wait for the page to reload
+              driver.getCurrentUrl should include(Welder.localSafeModeBaseDirectory)
+              notebookPage.getMode shouldBe NotebookMode.SafeMode
+            }
+          }
         }
       }
     }
+  }
 
+  // this test is important to make sure all components exit gracefully when their functionality is not needed
+  "User should be able to create files outside of playground and safe mode" ignore { runtimeFixture =>
```

Comment:
we changed this to `ignore`

Reply:
ah good catch, did this when playing around with one parallel suite before I added back tests